### PR TITLE
VZ-5503 First iteration of the new multicluster pipeline

### PR DESCRIPTION
**Pull request by pshirodkar-orcl**
# Description

First iteration of the new multicluster pipeline

Fixes VZ-5503

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/upgrade-paths/Jenkinsfile [hunk 1]
```diff
@@ -1,18 +1,30 @@
-// Copyright (c) 2022, Oracle and/or its affiliates.
+// Copyright (c) 2021, 2022, Oracle and/or its affiliates.
```
Comment (kminder) [inline ci/upgrade-paths/Jenkinsfile:1]: Why did you add 2021?

Comment (pshirodkar-orcl) [reply ci/upgrade-paths/Jenkinsfile]: I added it accidentally. I have removed it now.

**Review by kminder — DISMISSED** [review]
Approved.

**Review by Abhijithang — APPROVED** [review]
